stowaway.hcl

```diff
@@ -0,0 +1,17 @@
+description = "Symlink farm manager similar to GNU Stow"
+homepage = "https://github.com/jamesbehr/stowaway"
+binaries = ["stowaway"]
+// strip = 1
+source = "https://github.com/jamesbehr/stowaway/releases/download/v${version}/stowaway-${os}-${arch}.tar.gz"
+
+version "0.1.0" {
+  auto-version {
+    github-release = "jamesbehr/stowaway"
+  }
+}
+
+sha256sums = {
```

Comment:
`stowaway-linux-arm64.tar.gz` is missing -- https://github.com/jamesbehr/stowaway/releases/download/v0.1.0/stowaway-linux-arm64.tar.gz